packages/react/src/components/Masthead/__tests__/MastheadLeftNav.test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import MastheadLeftNav from '../MastheadLeftNav';
+import { mount } from 'enzyme';
+import React from 'react';
+import { TranslationAPI } from '@carbon/ibmdotcom-services';
+
+const { stablePrefix } = ddsSettings;
+
+describe('MastheadLeftNav', () => {
+  it('renders one MastheadLeftNav', () => {
+    const wrapper = mount(<MastheadLeftNav navigation={[]} />);
+    expect(wrapper).toHaveLength(1);
+  });
+
+  it('receives the `navigation` prop correctly', async () => {
+    const data = await TranslationAPI.getTranslation();
+    const wrapper = mount(
+      <MastheadLeftNav navigation={data.mastheadNav.links} />
+    );
+    expect(wrapper.prop('navigation')).not.toBeUndefined();
+  }, 10000);
```

Comment:
I think we can do the same here with passing mock data here

Reply:
@guilhermelMoraes can you take a look?

Reply:
yes, I will